Migration.md

```diff
@@ -0,0 +1,351 @@
+
+### COMPARE CREATE METHODS
+
+Here is a sample of a lob-node CREATE function:
+
+```javascript
+const Lob = require("lob")("YOUR API KEY");
+
+Lob.addresses.create(
+  {
+    name: "Thing T. Thing",
+    address_line1: "1313 CEMETERY",
+    address_city: "WESTFIELD",
+    address_state: "NJ",
+    address_zip: "07000",
+  },
+  function (err, res) {
+    console.log(err, res);
+  }
+);
+```
+
+Here is a sample of TypeScript SDK CREATE method:
+
+```typescript
+const addressCreate: AddressEditable = new AddressEditable({
+  name: "Thing T. Thing",
+  address_line1: "1313 CEMETERY LN",
+  address_line2: ""
+  address_city: "WESTFIELD",
+  address_state: "NJ",
+  address_zip: "07090",
+  address_county: "US"
+});
+try {
+  const myAddress = await new AddressesApi(config).create(addressCreate);
+} catch (err: any) {
+  console.error(err);
+}
+```
+
+If you are using the new TypeScript SDK using JavaScript the code is largely the same apart from removing the "type" identifiers found in TypeScript. Here is the code that you would use for this method:
+
+```javascript
+const addressData = new AddressEditable({
+  name: "Wednesday Addams",
+  address_line1: "1313 CEMETERY LN",
+  address_city: "WESTFIELD",
+  address_state: "NJ",
+  address_zip: "07090",
+});
+try {
+  const address = await new AddressesApi(config).create(addressData);
+} catch (err) {
+  console.error(err);
+}
+```
+
+### COMPARE LIST METHODS
+
+Here is a sample of a lob-node LIST method:
+
+```javascript
+Lob.addresses.list({ limit: 2 }, function (err, address) {
+  if (err) {
+    console.log(err);
+    return;
+  }
+});
+```
+
+Here is a sample of the TypeScript SDK LIST method:
+
+```typescript
+try {
+  const response = await new AddressesApi(config).list();
+} catch (err: any) {
+  console.error(err);
+}
+```
+
+### COMPARE GET BY ID METHOD
+
+Here is a sample of a lob-node GET method:
+
+```javascript
+Lob.addresses.retrieve("adr_xxxx", function (err, address) {
+  if (err) {
+    console.log(err);
+    return;
+  }
+});
+```
+
+Here is a sample of the TypeScript SDK GET by ID method:
+
+```typescript
+try {
+  const address = await new AddressesApi(config).get("adr_xxxx");
+} catch (err: any) {
+  console.error(err);
+}
+```
+
+### COMPARE DELETE METHOD
+
+​Here is a sample of the lob-node DELETE method:
+
+```javascript
+Lob.addresses.delete("adr_43769b47aed248c2", function (err, address) {
+  if (err) {
+    console.log(err);
+    return;
+  }
+});
+```
+
+Here is a sample of the TypeScript SDK DELETE method:
+
+```typescript
+try {
+  const deletedAddress = await new AddressesApi(config).delete("adr_xxxx");
```

Comment:
lets use the same id between the two samples. So either "adr_43769b47aed248c2" or "adr_xxxx"